api/tests/unit/domain/services/scoring/scoring-service_test.js

```diff
@@ -82,4 +82,19 @@ describe('Unit | Service | Scoring Service', () => {
     });
 
   });
+
+  describe('#totalUserPixScore', function() {
+
+    it('should return the sum of the floored earned pixScore by competence', () => {
+      // given
+      const pixEarnedByCompetences = [ 9.3, 50, 10 ];
```

Comment:
Proposition : 
```suggestion
      const pixEarnedByCompetences = [ 9.3, 50, 10.8 ];
```
Pour s'assurer qu'on ne prend pas en compte les nombres après la virgule.